docs/cep/reference/functions/adhoc/adhoc.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Adhoc Query
+---
+
+Adhoc queries provide a way of performing ad-hoc operations on tables (stores), named-windows, and named-aggregations. We can send adhoc queries and fetch data from Stores and named windows.
```

Comment:
```suggestion
Ad hoc queries provide a way of performing ad-hoc operations on tables (stores), named windows, and named aggregations. We can send ad hoc queries and fetch data from stores and named windows.
```

Reply:
Can you give a little more explanation of what an ad hoc query is in this context? Saying an ad hoc query lets me perform "ad hoc operations" doesn't tell me anything if I don't know what ad hoc either thing is.